content/en/blog/_posts/2021-03-09-Defining-NetworkPolicy-Conformance-For-CNIs

```diff
@@ -0,0 +1,474 @@
+---
+layout: blog
+title: "Defining Network Policy Conformance for Container Network Interface (CNI) providers"
+date: 2021-03-21
+slug: defining-networkpolicy-conformance-cni-providers
+---
+
+Authors: Matt Fenwick, Jay Vyas, Ricardo Katz, Amim Knabben, Douglas Schilling Landgraf
```

Comment:
We typically also credit the organizations that authors work for. Not mandatory though.

Reply:
done thanks